src/status_im2/common/floating_button_page/style.cljs

```diff
@@ -0,0 +1,15 @@
+(ns status-im2.common.floating-button-page.style)
+
+(def page-container
+  {:position :absolute
+   :top      0
+   :bottom   0
+   :left     0
+   :right    0})
+
+(def keyboard-avoiding-view
```

Comment:
I added this back in again as it's needed  to push the button to the bottom of screen to sit on top of the keyboard.
@ulisesmac we provided this solution to Erik some months ago for the create profile page